app/src/main/java/com/example/concordia_campus_guide/Helper/PathFinder.java

```diff
@@ -12,18 +15,16 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.Optional;
 import java.util.PriorityQueue;
-import java.util.function.Predicate;
 
 /**
  * Usage Example:
- * Double[] src = {-73.57883681, 45.49731974};
- * Double[] trg = {-73.57873723, 45.49748425};
- * RoomModel room1 = new RoomModel(src,"H927" ,"H-9");
- * RoomModel room2 = new RoomModel(trg,"H842", "H-8");
- * PathFinder finder = new PathFinder(getContext(),  room1, room2);
- * List<WalkingPoint> solution = finder.getPathToDestination();
+ Double[] src = {-73.5791637, 45.49526596};
```

Comment:
Are these still commented out?

Reply:
Yes, they are just an example usage for people to get an example fast fast.